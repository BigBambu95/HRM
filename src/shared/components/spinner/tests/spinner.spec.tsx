import React from 'react'
import { shallow } from 'enzyme'
import Spinner from '../spinner'

describe('Spinner', () => {
	it('Рендер спиннера', () => {
		const spinner = shallow(<Spinner />)
		expect(spinner).toMatchSnapshot()
	})
})
