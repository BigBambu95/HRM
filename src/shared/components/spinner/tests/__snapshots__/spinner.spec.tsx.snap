// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Рендер спиннера 1`] = `
<div
  className="spinner"
>
  <span />
  <span />
  <span />
  <span />
</div>
`;
